cli/run_util.go

```diff
@@ -220,13 +220,10 @@ func CreateBitriseConfigFromCLIParams(bitriseConfigBase64Data, bitriseConfigPath
 
 	isConfigVersionOK, err := versions.IsVersionGreaterOrEqual(models.FormatVersion, bitriseConfig.FormatVersion)
 	if err != nil {
-		log.Warn("bitrise CLI model version: ", models.FormatVersion)
-		log.Warn("bitrise.yml Format Version: ", bitriseConfig.FormatVersion)
-		return models.BitriseDataModel{}, warnings, fmt.Errorf("Failed to compare bitrise CLI models's version with the bitrise.yml FormatVersion: %s", err)
+		return models.BitriseDataModel{}, warnings, fmt.Errorf("Failed to compare bitrise CLI models's version (%s) with the bitrise.yml FormatVersion (%s): %s", models.FormatVersion, bitriseConfig.FormatVersion, err)
```

Comment:
NOTE: I was wondering if the Bitrise CLI model version has any meaning for users.
What do you think about something like this: `The bitrise.yml has a higher Format Version (12) than the bitrise CLI supported format version (11), please upgrade your bitrise CLI to use this bitrise.yml`

Reply:
That is a good suggestion. I have updated both error messages to remove the cli model version. This way it does seem more user friendly.